README.md

```diff
@@ -107,22 +107,11 @@ api.secret=${API_SECRET}
 ### How to run tests
 ### How to Checkstyle
 ### How to run data migration
-You can update DB with this CLI application. For that purpose you should run it from home-data-migration directory in two ways.
-1. Use full URL(--url) for connection to your DB
-   
-   `java -jar ./target/home-data-migration-0.0.1-SNAPSHOT.jar --url=jdbc:postgresql://localhost:5432/?user=user&password=password`
-   
-2. Use URL(--url), username(-u or --user) and password(-p or --password) for connection to your DB
+You can update DB with maven plugin. For that purpose you should run it in home-data-migration module by using commands:
```

Comment:
We have two options:

1. using maven build and enable profile, suitable for local needs
2. using app, suitable for real envs

Reply:
Got back old realization.